packages/truffle-environment/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "truffle-environment",
+  "version": "0.1.0",
+  "description": "This package contains the code needed to determine environment-specific data",
+  "main": "index.js",
+  "author": "Faina Shalts",
+  "license": "MIT",
+  "private": false,
+  "dependencies": {
+    "debug": "^4.1.0",
+    "ganache-core": "2.5.5",
+    "node-ipc": "^9.1.1",
+    "truffle-artifactor": "^4.0.19",
+    "truffle-error": "^0.0.5",
+    "truffle-expect": "^0.0.9",
+    "truffle-interface-adapter": "^0.1.6",
+    "truffle-resolver": "^5.0.14",
+    "truffle-workflow-compile": "^2.0.19",
```

Comment:
Not sure if we need `truffle-workflow-compile` added to the pkg.

Reply:
You're totally right. Removed.